src/Mechanic.fs

```diff
@@ -57,8 +61,10 @@ let activate (context : Vscode.ExtensionContext) =
                 state.Add project
         ) |> ignore
 
-        Vscode.commands.registerCommand("mechanic.run", fun _ ->
-            Promise.start runInScope
+        Vscode.commands.registerCommand("mechanic.run", fun input ->
+            let m = unbox<ProjectExplorerModel option> input
+            printfn "M: %A" m
```

Comment:
We probably should remove this log

Reply:
Woops

Reply:
will do

Reply:
Already done ;P

Reply:
awesome. and release process works! https://marketplace.visualstudio.com/items?itemName=Ionide.mechanic